specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/examples/SuccessfulEntitiesRequest.json

```diff
@@ -0,0 +1,94 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Endpoint": "{Endpoint}",
+    "input": {
+      "documents": [
+        {
+          "id": "1",
+          "language": "en",
+          "text": "Microsoft was founded by Bill Gates and Paul Allen."
+        },
+        {
+          "id": "2",
+          "language": "en",
+          "text": "Pike place market is my favorite Seattle attraction."
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "documents": [
+          {
+            "entities": [
```

Comment:
It may be helpful to add some resolutions to the examples to document the output, and also to ensure that the expected service output matches the spec during validation tests which run for each PR.